packages/web3-providers-http/types/index.d.ts

```diff
@@ -30,7 +30,6 @@ export interface HttpHeader {
 }
 
 export interface HttpProviderAgent {
-    baseUrl?: string;
```

Comment:
Have removed it since this isn't supported by fetch as well as this is not a standard for Http Agent